File: src/main/java/ankol/mod/merger/merger/scr/node/ScrLeafScriptNode.java
package ankol.mod.merger.merger.scr.node;

import ankol.mod.merger.core.BaseTreeNode;

/**
 * 叶子节点
 */
public class ScrLeafScriptNode extends BaseTreeNode {

    public ScrLeafScriptNode(String signature, int startTokenIndex, int stopTokenIndex, int line, String text) {
        super(signature, startTokenIndex, stopTokenIndex, line, text);
    }
}

File: src/main/java/ankol/mod/merger/merger/scr/node/ScrLeafScriptNode.kt
package ankol.mod.merger.merger.scr.node

import ankol.mod.merger.core.BaseTreeNode
import org.antlr.v4.runtime.TokenStream

/**
 * 叶子节点
 */
class ScrLeafScriptNode(
    signature: String,
    startTokenIndex: Int,
    stopTokenIndex: Int,
    line: Int,
    tokenStream: TokenStream
) : BaseTreeNode(signature, startTokenIndex, stopTokenIndex, line, tokenStream)
